services/movie_session.py

```diff
@@ -0,0 +1,53 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int) -> None:
+    movie_session = MovieSession.objects.create(show_time=movie_show_time)
+    create_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    create_movie = Movie.objects.get(id=movie_id)
+    movie_session.cinema_hall = create_hall
+    movie_session.movie = create_movie
+    movie_session.save()
```

Comment:
In the `create_movie_session` function, the `MovieSession` instance should be created with all necessary fields at once. Instead of creating the `MovieSession` with only `show_time` and then assigning `cinema_hall` and `movie` separately, you should pass `cinema_hall=create_hall` and `movie=create_movie` directly in the `create` method.